grpc-java/LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Ok - I think I'm going to copy these, so that they are readably in the grpc-java directory, and auto-picked up as the correct license for artifacts in each project.